pkg/rsyslog/init.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+mkdir -p /run/watchdog/pid 2>/dev/null || :
+(cd /run/watchdog/pid && touch logread.pid rsyslogd.pid monitor-rsyslogd.pid)
```

Comment:
This gives you 60 seconds to start logread hence the LOGREAD_PID_WAIT of 3600 doesn't make sense any more.

Reply:
I suppose being more conservative about this type of timing is good. Let me move those touches as close to the actual start of the process as possible. See the latest update @nordmark